src/script/page/MainContent/panels/preferences/DevicesPreferences/components/DetailedDevice.tsx

```diff
@@ -53,7 +53,11 @@ export const DetailedDevice: React.FC<DeviceProps> = ({
       </h3>
 
       {mlsFingerprint && (
-        <MLSDeviceDetails fingerprint={mlsFingerprint} isOtherDevice={isOtherDevice} certificate={certificate} />
+        <MLSDeviceDetails
+          fingerprint={mlsFingerprint}
```

Comment:
is this the same fingerPrint that you are not getting from corecrypto network call directly, a comment you made above?

Reply:
Here we are dealing with the self user's devices. In this case we get the fingerprint from backend. 

But for someone else's device, this info is not returned by backend. I'll need to investigate how to get that info for those devices